clients/src/test/java/org/apache/kafka/clients/admin/MockAdminClient.java

```diff
@@ -243,6 +245,8 @@ private MockAdminClient(
         this.brokerLogDirs = brokerLogDirs;
         this.brokerConfigs = new ArrayList<>();
         this.clientMetricsConfigs = new HashMap<>();
+        this.groupConfigs = new HashMap<>();
+        this.defaultGroupConfigs = generateDefaultGroupConfigs();
```

Comment:
Should we rather define a constant for the default group configs? Thinking a bit more about those too. I wonder if we should rather extend the builder to let the user passes the default group config that he wants to use instead of having some defaults defined here. Then, we can have a constant on the user side which is a litter clearer in my opinion because you see them when you read the tests. What do you think?

Reply:
Will do, thanks for the suggestion.